README.md

```diff
@@ -121,6 +121,19 @@ After generating a new version you should commit the changes. This will be the C
 
 Note: The Changelog will be regenerated with all tags on release on the external builder tool, but will not be checked in there. The Changelog on the github release will therefore always be correct, on the repo it might be incorrect due to missing tags when executing the `yarn release` command.
 
+## How the different .env work on deploy
+
+Each component (frontend, admin, backend and database) has its own `.env` file. When running in development you usually do not have to care about the `.env`. The defaults are set by the respective config file, found in the `src/config/` folder of each component. But if you have a local `.env`, the defaults set in the config are overwritten by the `.env`.
```

Comment:
...When running in development you usually do not have to care about the `.env`....
This statement seems confusing, because without any .env in the components directories, they will not work together:
- e.g. the frontend will not know the port of the backend

Is there a automatic process, which creates o manipulates the .env-Files?

Reply:
That is wrong.
```
const endpoints = {
  GRAPHQL_URI: process.env.GRAPHQL_URI || 'http://localhost/graphql',
  ADMIN_AUTH_URL: process.env.ADMIN_AUTH_URL || 'http://localhost/admin/authenticate?token={token}',
}
```

Reply:
The frontend knows where the backend is

Reply:
As you say in the frontend src/config/index.ts the property
  GRAPHQL_URI: process.env.GRAPHQL_URI || 'http://localhost/graphql',
is defined.
A running frontend base on this configuration will not find the running backend, because in the backend/src/config/index.ts the property
  PORT: process.env.PORT || 4000,
is defined.
This means, the backend supports the URL http://localhost:4000/graphql and the frontend will not find the backend by using the configured URL http://localhost/graphql.
Either the default configurations of frontend and backend must be changed to match the GRAPHQL_URL or a local .env must be defined to correct the GRAPHQL_URL.